src/main/php/PDepend/Util/Cache/Driver/MemoryCacheDriver.php

```diff
@@ -98,7 +98,7 @@ class MemoryCacheDriver implements CacheDriver
      */
     public function __construct()
     {
-        $this->staticId = sha1(uniqid((string) rand(0, PHP_INT_MAX)));
+        $this->staticId = sha1(uniqid((string) mt_rand(0, PHP_INT_MAX)));
```

Comment:
```suggestion
        $this->staticId = sha1(uniqid((string) mt_rand()));
```
0 is the default and [mt_getrandmax()](https://www.php.net/manual/fr/function.mt-getrandmax.php) is more relevant theoretically.

On the other side it sounds like we just want a random hexadecimal string so it would be better to do:
```suggestion
        $this->staticId = bin2hex(random_bytes(20));
```

(20 bytes is equivalent length of sha1)

Reply:
Good thinking